.github/CODEOWNERS

```diff
@@ -386,25 +386,25 @@ lerna.json @grafana/frontend-ops
 /public/app/plugins/panel/alertGroups/ @grafana/alerting-squad-frontend
 /public/app/plugins/panel/alertlist/ @grafana/alerting-squad-frontend
 /public/app/plugins/panel/annolist/ @grafana/user-essentials
-/public/app/plugins/panel/barchart/ @grafana/grafana-bi-squad
+/public/app/plugins/panel/barchart/ @grafana/grafana-bi-squad @grafana/dataviz-squad
 /public/app/plugins/panel/bargauge/ @grafana/user-essentials
 /public/app/plugins/panel/dashlist/ @grafana/user-essentials
 /public/app/plugins/panel/debug/ @ryantxu
 /public/app/plugins/panel/gauge/ @grafana/user-essentials
```

Comment:
While we are making these changes, we might as well include the items we are also taking from user-essentials:
- `gauge`
- `bargauge`
- `stat`

Reply:
@nmarrs do you know if the edge-squad GitHub handle is going to remain?

Reply:
It will be removed -> have [a PR up for that](https://github.com/grafana/grafana/pull/62408) but we probably could just change everything `edge` to `dataviz` in this PR and close mine

Reply:
I changed it here.